docs/data/hubble/analyst-guide/queries-for-horizon-like-data.mdx

```diff
@@ -0,0 +1,922 @@
+  from assets
+    join `crypto-stellar.crypto_stellar.claimable_balances` as cb
+      on assets.asset_code = cb.asset_code
+      and assets.asset_issuer = cb.asset_issuer
+  where true
+  order by last_modified_ledger asc
+)
+select
+  *
+from current_claimable_balances
+where true
+  and rn = 1
+  and deleted = false
+  and balance_id = <claimable balance id>
+```
+
+### Retrieve a Related Operations and Transactions
+
+```sql
+select
+  op_id
+  , transaction_id
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.enriched_history_operations`
+where true
+  and balance_id = <claimable_balance_id>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+## Effects
+
+[horizon/effects](/docs/data/horizon/api-reference/resources/effects)
+
+Effects represent specific changes that occur in the ledger as a result of successful operations, but are not necessarily directly reflected in the ledger or history, as transactions and operations are.
+
+### List All Effects
+
+```sql
+select
+  type
+  , type_string
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.history_effects`
+where true
+  -- Optional filtering options
+  --and operation_id = <operation id>
+  --and address = <account address>
+  --and details.liquidity_pool_id = <liquidity pool id>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+## Ledgers
+
+[horizon/ledgers](/docs/data/horizon/api-reference/resources/ledgers)
+
+Each ledger stores the state of the network at a point in time and contains all the changes - transactions, operations, effects, etc. - to that state.
+
+Learn more about [ledgers](../../../learn/glossary.mdx#ledger).
+
+### List All Ledgers
+
+```sql
+select
+  sequence
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar.history_ledgers`
+where true
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+### Retrieve a Ledger’s Transactions and Operations
+
+```sql
+select
+  ledger_sequence
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.enriched_history_operations`
+where true
+  and ledger_sequence = <ledger sequence>
+  -- highly recommended to provide a date range to reduce amount of data queried
+  and closed_at between <start> and <end>
+```
+
+### Retrieve a Ledger’s Payments
+
+```sql
+select
+  ledger_sequence
+  -- please see table schema for more available columns and data
+from `crypto-stellar.crypto_stellar_dbt.enriched_history_operations` as eho
+where true
+  and (type = 0 OR type = 1 OR type = 8 OR (type = 2 OR type = 13 AND (eho.to != eho.from)))
```

Comment:
you probably want to keep payments where `to=from` it's still a valid payment from the network perspective

Reply:
(unless horizon filters them out that way too)

Reply:
Will remove
Horizon does not filter this out https://stellarfoundation.slack.com/archives/C02B04RMK/p1719950212626139